DQMOffline/EGamma/plugins/ElectronTagProbeAnalyzer.cc

```diff
@@ -665,11 +665,11 @@ bool ElectronTagProbeAnalyzer::isolationCut(const reco::GsfElectronCollection::c
 
   if (gsfIter->dr03TkSumPt() > tkIso03Max_)
     return true;
-  if (gsfIter->isEB() && gsfIter->dr03HcalDepth1TowerSumEt() > hcalIso03Depth1MaxBarrel_)
+  if (gsfIter->isEB() && gsfIter->dr03HcalTowerSumEt(1) > hcalIso03Depth1MaxBarrel_)
```

Comment:
in Run3 HCAL now have more than one depth, so you should have the same multiple cases as in the isEE() ?
This seems true for all the DQMOffline instances

Reply:
Yes, and this is deliberate. This PR is intended purely for the tower to rechit migration, any other code is updated only to the extent necessary to be able to compile and run cmssw.

The fact that the meaning of things monitored have changed, or whether to monitor individual depths are outside of the scope and should be addressed elsewhere.

Reply:
Could this be a follow-up PR in DQM?